src/_helpers/auth.ts

```diff
@@ -0,0 +1,92 @@
+import jwt from 'jsonwebtoken';
+import { logger } from "../index";
+import { currentConfig } from './config';
+import { clone } from "./clone";
+
+import { AuthenticateSuccessResponse } from '../_models/AuthenticateSuccessResponse';
+import { User } from '../_models/User';
+
+export function authenticate(username: string, password: string): Promise<AuthenticateSuccessResponse> {
+    return new Promise<AuthenticateSuccessResponse>((resolve, reject) => {
+        console.log(currentConfig.users);
```

Comment:
Can we remove this debug line?
Passwords (more on that in another comment) are printed as plain text visible for everyone who has access to the logs...